README.md

```diff
@@ -16,11 +16,16 @@ Deploy [Prometheus](https://github.com/prometheus/prometheus) monitoring system
 
 When upgrading from <= 2.4.0 version of this role to >= 2.4.1 please turn off your prometheus instance. More in [2.4.1 release notes](https://github.com/cloudalchemy/ansible-prometheus/releases/tag/2.4.1)
 
+### Install with docker
+
+You can install prometheus through docker. Related variables are prefixed with `prometheus_docker` and are explained below.
+
 ## Requirements
 
 - Ansible >= 2.5 (It might work on previous versions, but we cannot guarantee it)
 - jmespath on deployer machine. If you are using Ansible from a Python virtualenv, install *jmespath* to the same virtualenv via pip.
 - gnu-tar on Mac deployer host (`brew install gnu-tar`)
+- If prometheus is installed with docker, docker must be installed prior to any deployment with this role, you cand use [this role](https://galaxy.ansible.com/geerlingguy/docker).
```

Comment:
Please change role to https://github.com/mongrelion/ansible-role-docker. This is purely for marketing reasons :)